runtime/oti/j2sever.h

```diff
@@ -52,13 +61,23 @@
 /*
  * Note: J2SE_SHAPE_LATEST has to be updated to highest JCL level supported by VM
  * 		 This allows JVM operates with latest level when classlib.properties doesn't present.
+ * Additional note:
+ * 		 J2SE_SHAPE_LATEST will be kept as J2SE_SHAPE_B165 while Java 18.3 raw build is being built.
+ * 		 When a separate Java 18.3 binary is required, a #ifdef flag will be introduced to define
+ * 		 J2SE_SHAPE_B165 to different values accordingly such as following:
+ * 		 #if J9VM_JAVA9_BUILD > 9
+ * 		 	#define J2SE_SHAPE_LATEST  J2SE_SHAPE_B1803
+ * 		 #else
+ * 		 	#define J2SE_SHAPE_LATEST  J2SE_SHAPE_B165
+ * 		 #endif
  */
-#define J2SE_SHAPE_LATEST    	J2SE_SHAPE_B165
+#define J2SE_SHAPE_LATEST  		J2SE_SHAPE_B165
 #define J2SE_SHAPE_SUN     		0x10000
 #define J2SE_SHAPE_B136    		0x40000
 #define J2SE_SHAPE_B148    		0x50000
 #define J2SE_SHAPE_B165    		0x60000
-#define J2SE_SHAPE_RAWPLUSJ9	0x80000
+#define J2SE_SHAPE_B1803   		0x70000
+#define J2SE_SHAPE_RAWPLUSJ9   	0x80000
```

Comment:
There is a mix of spaces and tabs here that won't leave everyone pleased. IMO tabs should never appear after a non-tab.

Reply:
In particular, we try to avoid this kind of alignment as keeping the spacing consistent when new entries are added results in unnecessary changes.

Reply:
Removing the space changes to other entries to avoid unnecessary modification.